hc/api/management/commands/sendalerts.py

```diff
@@ -43,6 +46,14 @@ def handle_one(self, check):
         # Save the new status. If sendalerts crashes,
         # it won't process this check again.
         check.status = check.get_status()
+
+        if check.status is 'up':
+            check.nag_mode = False
+        else:
```

Comment:
In the case that a check has any other status besides "up", i.e "paused", "new" or "down" the `nag_mode` is going to be set to `True`. Correct me if I am wrong, but that ought to happen only when the status is "down".

Be explicit about that!

Reply:
@collinmutembei only the checks that are up or down are passed to this function. This covers us from having to filter those that are "paused" or "new"

Reply:
Ah! got it